library/core/src/ffi/primitives.rs

```diff
@@ -0,0 +1,63 @@
+//! Platform-specific types, as defined by C.
+//!
+//! Code that interacts via FFI will almost certainly be using the
+//! base types provided by C, which aren't nearly as nicely defined
+//! as Rust's primitive types. This module provides types which will
+//! match those defined by C, so that code that interacts with C will
+//! refer to the correct types.
+//!
+//! Contains only the primitive types, mod.rs contains additional types
+
+#![stable(feature = "core_ffi", since = "1.30.0")]
```

Comment:
Same here, attribute shouldn't be needed.

For the doc comment, could you mention that this only contains type definitions to keep it easy to parse externally?